tools/SourceKit/tools/sourcekitd/lib/Service/Requests.cpp

```diff
@@ -938,7 +938,7 @@ handleRequestEditorOpenInterface(const RequestDict &Req,
     std::optional<StringRef> InterestedUSR = Req.getString(KeyInterestedUSR);
     return Rec(editorOpenInterface(*Name, *ModuleName, GroupName, Args,
                                    SynthesizedExtension, InterestedUSR));
-  }
+  });
 }
 
 static void handleRequestEditorOpenHeaderInterface(
```

Comment:
While here, should we also do the same for header interfaces?

Reply:
Let’s only change this request for now. We don’t use the open header interface on Windows (ie. from SourceKit-LSP) and it hasn’t been an issue on macOS. At some point we should probably re-think what queues all the callbacks etc. run on and whether they need a deep stack…